jordan_bundy/models/caffeine.js

```diff
@@ -0,0 +1,12 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var caffeineSchema = Schema({
+    _id: Number,
+    type: String,
+    units: Number,
+    consumer: [{type: Number, ref: 'Developr'}]
```

Comment:
Nope, that was a typo, but since I didn't end up using that reference I didn't catch it!